extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -1590,6 +1602,20 @@ namespace Microsoft.Windows.DevHome.SDK
         Windows.Foundation.IAsyncOperationWithProgress<QuickStartProjectResult, QuickStartProjectProgress> GenerateAsync();
     }
 
+    // Extensions implement this and is provided to Dev Home when CreateChatStyleGenerationOperation is called.
```

Comment:
`Extensions implement this` yes.  They do.  That is know in the SDK.  It can be removed.

Reply:
That makes sense. I moved some of the comments around and wrote a better comment for GenerateChatStyleResponse().